### ISSUE TITLE
Remove issues from inbox/done if you're not longer assigned

### ISSUE THREAD

**Issue by fluteds** [isaacs/github#1929]
#### Summary 
Update the notifications/[done tab](https://github.com/notifications?query=is%3Adone) if you're no longer assigned to an issue.
If GitHub really wants to keep history of assigned issues maybe create another tab with previous assigned history or add the feature to remove/delete marked done issues completely. 
![image](https://user-images.githubusercontent.com/34608301/109146271-86122f00-775b-11eb-9b9e-95ce1389bebe.png)

#### Why does this need to be changed? 

1. Clogs up workflow as you can't see what you've actually been assigned if you ever need to go back to a closed issue and it's not been saved.
2. Really annoying if you use issues for monthly/daily summaries. 

**TL;DR**

- When a user is no longer assigned to an issue, remove the issue from the done tab.
- Add another tab if GitHub really wants to keep assigned issue history. **e.g.** Assigned Issue History
- This cleans up the done tab as when you're no longer involved it just clogs up what you have been assigned and worked on.

Attached this issue and sent it to the feedback site @ [support.github.com/contact/feedback.](https://support.github.com/contact/feedback)
